apps/condo/domains/news/schema/NewsItemScope.test.js

```diff
@@ -292,29 +314,11 @@ describe('NewsItemScope', () => {
             )
         })
 
-        test('must throw an error on trying to create empty scope', async () => {
-            const [newsItem] = await createTestNewsItem(adminClient, dummyO10n)
-            await expectToThrowGQLError(
-                async () => await createTestNewsItemScope(adminClient, newsItem, { newsItem: null }),
-                {
-                    code: 'BAD_USER_INPUT',
-                    type: 'EMPTY_NEWS_ITEM_SCOPE',
-                    message: 'News item scope is empty',
-                    messageForUser: 'api.newsItem.EMPTY_NEWS_ITEM_SCOPE',
-                },
-            )
-        })
-
         test('must throw an error on trying to create scope with unitName and without unitType', async () => {
             const [newsItem] = await createTestNewsItem(adminClient, dummyO10n)
-            await expectToThrowGQLError(
+            await expectToThrowValidationFailureError(
                 async () => await createTestNewsItemScope(adminClient, newsItem, { unitName: '1' }),
-                {
-                    code: 'BAD_USER_INPUT',
-                    type: 'UNIT_NAME_WITHOUT_UNIT_TYPE',
-                    message: 'You set unitName without unitType',
-                    messageForUser: 'api.newsItem.UNIT_NAME_WITHOUT_UNIT_TYPE',
-                },
+                'Required field "type" is null or undefined.',
```

Comment:
it looks like wrong error name

Reply:
Error details are ok. The test name was wrong. Fixed.